package.json

```diff
@@ -114,7 +117,8 @@
     "verbose": true,
     "moduleNameMapper": {
       "\\.(jpg|jpeg|png|gif|eot|otf|webp|svg|ttf|woff|woff2|mp4|webm|wav|mp3|m4a|aac|oga)$": "<rootDir>/__mocks__/fileMock.js",
-      "\\.(css|less|scss)$": "identity-obj-proxy"
+      "\\.(css|less|scss)$": "identity-obj-proxy",
+      "^@webex/component-adapter-interfaces(.*)$": "<rootDir>/node_modules/@webex/component-adapter-interfaces/dist/webex-component-adapter-interfaces.umd"
```

Comment:
Is this still required after the change in rollup config?

Reply:
This change is the one which fixes the module resolution error

Reply:
While running tests, jest started to fail with module resolution error so added moduleNameMapper to fix that